component/useLongPress.js

```diff
@@ -0,0 +1,63 @@
+// See https://stackoverflow.com/a/48057286, credits "sudo bangbang"
+
```

Comment:
Add to LICENSE that files in `licensed-components` are under separate licences, currently `CC BY-SA 4.0`.